pyproject.toml

```diff
@@ -42,7 +42,7 @@ testfixtures = "6.0.0"
 [tool.poetry.scripts]
 lint = 'scripts:lint'
 test = 'scripts:test'
-test-parser = 'test_parser:test_parser'
+test_parser = 'test_parser:test_parser'
```

Comment:
Is it worth keeping both to avoid confusing long-time contributors? :)
We should probably also update the wiki here: https://github.com/tmrowco/electricitymap-contrib/wiki/Building-a-new-parser#test-parsers-locally

Reply:
poetry run test-parser is not in the wiki. We mention `python test_parser.py` in a poetry shell.

Reply:
It's not mentioned in the wiki, but we have mentioned it in a lot of comments and issues :)

Reply:
I see. I'll add an alias here. :)